### PR TITLE
ERMS-2164

### PR DESCRIPTION
**Pull request by Inselhopper**

### CODE REVIEW

File: app/grails-app/views/organisation/show.gsp [hunk 1]
```diff
@@ -177,7 +177,7 @@
 
                                         <div class="fields">
                                             <div class="field">
-                                                <input name="value" id="value" type="text" class="ui" />
+                                                <input name="value" id="identificator" type="text" class="ui" />
```
Comment (croqueGrec09) [inline app/grails-app/views/organisation/show.gsp]: Should not be identifier the better convenient? I remark it for the sage of bulk search ...

Comment (Inselhopper) [reply app/grails-app/views/organisation/show.gsp]: Yes, that's right. It sounded odd.

File: app/grails-app/views/address/_formModal.gsp [hunk 2]
```diff
@@ -180,7 +179,7 @@
 </semui:modal>
 <r:script>
         function handleRequired() {
-            $('#create_address')
+            $("form[id*='create_address']")
```
Comment (klodav) [inline app/grails-app/views/address/_formModal.gsp]: $("form[id^='create_address']")

File: app/grails-app/views/dev/frontend.gsp [hunk 3]
```diff
@@ -469,12 +469,10 @@ if (customerType) {
         class="ui circular label la-js-filter-total hidden">0</span>
 </button>
 <h3>Icons in einem Link ohne zusätzlichen Text</h3>
-<g:link controller="yoda" action="frontend" params="" class="ui icon positive button">
-    <i  class="checkmark icon"></i>
-</g:link>
-<g:link controller="yoda" action="frontend" params="" class="ui icon negative button">
-    <i aria-hidden="true" class="times icon"></i>
+<g:link aria-label="Das ist eine Beschreibung für den Accessibility Tree" controller="yoda" action="frontend" params="" class="ui icon positive button">
```
Comment (klodav) [inline app/grails-app/views/dev/frontend.gsp]: controller="dev"

**Pull request merged**